packages/browser/src/index.ts

```diff
@@ -57,6 +57,9 @@ export {
   BrowserTracing,
   defaultRequestInstrumentationOptions,
   instrumentOutgoingRequests,
+  browserTracingIntegration,
+  startBrowserTracingNavigationSpan,
```

Comment:
These two APIs would be used by custom instrumentation, e.g. Angular, Ember, Next etc.